spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -116,6 +116,8 @@ module MiqAeServiceSpec
           Datacenter
           ManageIQ::Providers::BaseManager
           ManageIQ::Providers::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::Provider
```

Comment:
Either way is fine.  I just find it odd that we modifying the test to avoid failures and spending time discussing this when it should not take more than a few mins to add the missing service models in the first place.